HucaresWF/src/Hucares.Server.Client/DetectedPlateClient.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Hucares.Server.Client.Models;
+using Newtonsoft.Json;
+
+namespace Hucares.Server.Client
+{
+    public class DetectedPlateClient : IDetectedPlateClient
```

Comment:
Class and interface not added to project compile

Reply:
Added to .csproj